### PR TITLE
add dependencies for rocm5.0

### PR DESCRIPTION
**Pull request by KyleCZH**
Signed-off-by: Kyle Chen <kylechen@amd.com>

@jithunnair-amd

### CODE REVIEW

File: manywheel/build_common.sh [hunk 1]
```diff
@@ -228,7 +228,9 @@ fname_with_sha256() {
     BASENAME=$(basename $1)
     # Do not rename nvrtc-builtins.so as they are dynamically loaded
```
Comment (jithunnair-amd) [inline manywheel/build_common.sh:229]: Please move this comment to the elif and add a comment for ROCm here

**Comment by jithunnair-amd** [comment]
@malfet ROCm5.0 base docker images are now released, can you please retrigger CI or let me know how to? Thanks!

**Comment by malfet** [comment]
@jithunnair-amd I just clicked on "Re-run workflow" button, see results in https://github.com/pytorch/builder/actions/runs/1808566987

**Comment by jithunnair-amd** [comment]
@KyleCZH We need to add the amdgpu version "21.50" for ROCm5.0 here: https://github.com/pytorch/builder/blob/main/common/install_rocm.sh#L25

Reference: https://github.com/ROCmSoftwarePlatform/pytorch/commit/e503deb4b31542457e3810820c9311e49285d309

**Review by seemethere — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Comment by malfet** [comment]
@jithunnair-amd is this PR ready to be merged?

**Comment by jithunnair-amd** [comment]
@malfet Yes, it is! :)

**Review by jithunnair-amd — APPROVED** [review]

**Pull request merged**